### PR TITLE
Corrected script for some item

### PR DESCRIPTION
**Pull request by idk-whoami**
* **Addressed Issue(s)**: 

* **Server Mode**: re

* **Description of Pull Request**:

### CODE REVIEW

File: db/re/item_db_equip.yml [hunk 1]
```diff
@@ -72624,6 +72748,36 @@ Body:
...
+    Weight: 1000
+    Attack: 140
+    MagicAttack: 170
+    Range: 5
+    Slots: 3
+    Jobs:
+      All: true
+      BardDancer: false
+      Hunter: false
+      Rogue: false
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks !

File: db/re/item_db_etc.yml [hunk 2]
```diff
@@ -9743,7 +9743,7 @@ Body:
     Buy: 20
     Weight: 10
     Locations:
-      Right_Hand: true
+      Armor: true
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_db_etc.yml]: Sorry saw that the iRO description has been updated

File: db/re/item_db_equip.yml [hunk 3]
```diff
@@ -72624,6 +72753,35 @@ Body:
...
+    SubType: Bow
+    Weight: 1000
+    Attack: 140
+    MagicAttack: 170
+    Range: 5
+    Slots: 3
+    Jobs:
+      BardDancer: false
+      Hunter: false
+      Rogue: false
```
Comment (Atemo) [inline db/re/item_db_equip.yml:72769]: There is still an issue here, it should be true
```suggestion
      BardDancer: true
      Hunter: true
      Rogue: true
```

Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks !

**Comment by RagnaWay** [comment]
add this 23049, 23050, 31818 too

**Comment by laziem** [comment]
Hi, can you add this too (db_etc): 300088
This `bonus2 bSkillAtk,"SJ_FALLINGSTAR_ATK",15*.@r;
      bonus2 bSkillAtk,"SJ_FULLMOONKICK",15*.@r;`

should be

`bonus2 bSkillAtk,"SJ_FALLINGSTAR_ATK",15;
      bonus2 bSkillAtk,"SJ_FULLMOONKICK",15`

**Comment by RagnaWay** [comment]
More missing item:
```
20559
19104
22245
31866
31867
31868
31869
31870
31871
31872
31873
31957
410051
450158
450181
470047
```

**Comment by Atemo** [comment]
That's a lot!
@idk-whoami do you want to add them? otherwise I merge this PR to master

**Comment by Haydrich** [comment]
410016 - 410017
change it to middle headgear

**Comment by Atemo** [comment]
Thanks!

**Pull request merged**